jane/doc/extensions/_05-modes/reference.md

```diff
@@ -47,3 +47,50 @@ can construct the lazy value at `many` even if the thunk is `once` (e.g., closin
 let r = { x = 0 } in
 let l @ many = lazy (overwrite_ r with { x = 42 })
 ```
+
+# Exceptions
+
+The exception type `exn` crosses portability and contention.
+
+For backwards compatibility with OCaml, we don't require exception constructor argument
+types to cross portability and contention themselves. Instead, we treat each instance
+of an exception constructor as belonging to the capsule it originally was defined in.
```

Comment:
We should mention the current soundness bug: they can be packed in a first-class module and sent to other threads.

Also, while I appreciate the capsule analogy, I think we need to either be more precise (by at least pointing to `project` and `inject` and explain the correspondence), or just remove the analogy and state the concrete behavior.

Reply:
Done.